src/Dapr.AspNetCore/DaprAuthorizationOptionsExtensions.cs

```diff
@@ -0,0 +1,29 @@
+﻿// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+namespace Microsoft.AspNetCore.Authorization
+{
+    using Dapr.AspNetCore;
+
+    /// <summary>
+    /// Provides extension methods for <see cref="AuthorizationOptions" />.
+    /// </summary>
+    public static class DaprAuthorizationOptionsExtensions
+    {
+        /// <summary>
+        /// Adds Dapr authorization policy.
+        /// </summary>
+        /// <param name="options">The <see cref="AuthorizationOptions"/>.</param>
+        /// <param name="name">The name of the policy.</param>
+        public static void AddDapr(this AuthorizationOptions options, string name = "Dapr")
```

Comment:
Is this policy name just descriptive?

Reply:
The name needs to match the policy used when adding `AuthorizeAttribute`. E.g. `[Authorize("Dapr")]`